src/app/mod.rs

```diff
@@ -1227,6 +1228,42 @@ impl<'a, 'b> App<'a, 'b> {
         self.get_matches_from_safe(&mut env::args_os())
     }
 
+    /// Similar to [`App::get_matches`] but also reads args from
+    /// env-var `env_var_name`.
+    ///
+    /// If the same argument is specified both as program argument
```

Comment:
Let's change this to say, "The env var args are appended to the manually entered arguments (if any) and thus if an argument is provided both in the env var *and* manually, this could result in either a conflict, override, or additional value depending on the specific rules for said arg."